src/Doctrine/Instantiator/Instantiator.php

```diff
@@ -31,37 +31,33 @@ final class Instantiator implements InstantiatorInterface
      *
      * @deprecated This constant will be private in 2.0
```

Comment:
you forgot to remove that `@deprecated` tag

Reply:
Right. Wanna submit a PR?